evennia/accounts/accounts.py

```diff
@@ -687,6 +687,7 @@ def create_character(self, *args, **kwargs):
             ip=character_ip,
             typeclass=character_typeclass,
             permissions=character_permissions,
+            location=ObjectDB.objects.get_id(settings.START_LOCATION),
```

Comment:
While the idea is good, this is an incorrect fix and cannot be merged as-is. The reason is that `**kwargs` may well contain the `location=<location>` keyword and that should override the default. 

The correct fix of this would be something like this before the object creation (e.g. on line 682):

```python

if 'location' not in kwargs:
    kwargs['location'] = ObjectDB.objects.get_id(settings.START_LOCATION)

```

Reply:
Thanks, I made your suggested change. At least, I think I did, I'm new to GitHub.